cla_backend/apps/cla_eventlog/tests/test_find_and_delete_old_cases.py

```diff
@@ -0,0 +1,196 @@
+from django.test import TestCase
+from django.utils import timezone
+from freezegun import freeze_time
+import datetime
+
+from core.tests.mommy_utils import make_recipe
+from cla_eventlog.models import Log
+from complaints.models import Complaint
+from legalaid.models import Case, EODDetails, EligibilityCheck, PersonalDetails
+from diagnosis.models import DiagnosisTraversal
+from cla_auditlog.models import AuditLog
+from cla_eventlog.management.commands.find_and_delete_old_cases import Command
+
+
+def _make_datetime(year=None, month=None, day=None, hour=0, minute=0, second=0):
+    today = datetime.date.today()
+    year = year if year else today.year
+    month = month if month else today.month
+    day = day if day else today.day
+    dt = datetime.datetime(year, month, day, hour, minute, second)
+    return timezone.make_aware(dt, timezone.get_current_timezone())
+
+
+class FindAndDeleteOldCases(TestCase):
+    def setUp(self):
+        super(FindAndDeleteOldCases, self).setUp()
+        self.instance = Command()
```

Comment:
Could you rename this variable, please?
Because I'd been reading the command previously, when reading your testI was convinced `instance` was the task...

I think `self.command` would be clearest all around

Reply:
Yep